hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/CommonUpgradeResources.java

```diff
@@ -29,7 +29,7 @@
 
 public final class CommonUpgradeResources {
     public static final String DEFAULT_UPGRADE_FILE_ID = "0.0.150";
-    public static final String DEFAULT_UPGRADE_FILE_PATH = "upgrade.zip";
+    public static final String DEFAULT_UPGRADE_FILE_PATH = "testfiles/poeticUpgrade.zip";
```

Comment:
Why is the filename changed here? Does it has any impact on other places?

Reply:
The old upgrade.zip file was deleted quite awhile ago during a reorganization of the test directory structure. I don't know whether it was deleted accidentally or on purpose, but no file with that name currently exists anywhere in the project. All I'm doing here is changing the test to use a different file which does still exist.

Reply:
This value is also used in PerfTestLoadSettings. I am not familiar with what this setting is used for.

Reply:
In PerfTestLoadSettings, DEFAULT_UPGRADE_FILE_PATH is never read except to output it in a toString() method. The way it was before (pointing to a non-existent file upgrade.zip) would have been broken. I think it's safe to change it here. It was definitely broken before.